src/utils/utils-contour.js

```diff
@@ -61,7 +65,7 @@ export const buildContourSQL = (state, mapBounds, isPolygons = false) => {
   const contourParamsSQL = buildParamsSQL(contourParams)
 
   const geometryColumn = isPolygons ? "contour_polygons" : "contour_lines"
-  const contourLineCase = `CASE mod(cast(contour_values as int), ${major_contour_interval}) WHEN 0 THEN 1 ELSE 0 END as is_major `
+  const contourLineCase = `CASE mod(cast(contour_values as int), ${major_contour_interval}) WHEN 0 THEN 1 ELSE 0 END as ${encoding.color.field} `
```

Comment:
utlimately I feel like including the CASE and such should be optional depending on whether the user is wants major/minor interval distinction or not.

Reply:
If you're OK w it i'm going to leave the CASE in there for now. This handles making distinctions in the styles and not, albeit not as efficiently if the user makes major/minor settings the same.